build.xml

```diff
@@ -10,19 +11,30 @@
 
 	<property name="version" value="0.0.0" />
 
-	<property name="build.dir" value="./bin" />
-	<property name="build.dir.compile" value="${build.dir}/classes" />
+	<property name="build.dir" value="./bin/main" />
+	<property name="build.dir.compile" value="${build.dir}/" />
 	<property name="build.dir.resources" value="${build.dir.compile}" />
 
-	<property name="test.build.dir" value="./bin"/>
-	<property name="test.build.dir.report" value="${test.build.dir}/report" />
-	<property name="test.build.dir.compile" value="${test.build.dir}/classes" />
+	<!-- Tests -->
+	<property name="test.groups" value="" />
+	<property name="test.excludegroups" value="long-runtime" />
```

Comment:
Do we have long running tests?  Tests that aren't run offer limited value as they won't be run early enough to prevent anything.  I realize with a downloader there will be some slow tests, but tweaking them to be faster might be worthwhile.

Now... if the long running tests ARE run on the server but not tweakable locally, that I am totally ok with.  I just don't want some tests to be runnable by humans first.